modules/compliance-custom-node-pools.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+:_content-type: PROCEDURE
+[id="compliance-custom-node-pools"]
+= Creating custom node pools
```

Comment:
This section might be called Scanning custom node pools or similar (we're not creating them here..)